### ISSUE TITLE
Plain JS object as apiSpec in constructor

### ISSUE THREAD

**Issue by sheldhur** [cdimascio/express-openapi-validator#9]
Hi. Can you add support plain JS object as apiSpec in constructor?

It would be useful if you use `jsdoc` annotations for writing swagger\openapi spec or if you split yor spec on small simpled parts (js or json) and build big yaml when app starts.

**Comment by cdimascio** [comment]
hi @sheldhur thanks for the suggestion. could you elaborate a bit on what you are looking for? perhaps provide an example of what you are thinking. i'll certainly consider. thanks!

**Comment by sheldhur** [comment]
@cdimascio like this

```js
const apiSpec = {
  openapi: "3.0.0",
  info: {...},
  servers: [...],
  paths: {...},
  components: {
    responses: {...},
    schemas: {...}
  }
}
new OpenApiValidator({apiSpec}).install(app);
```

**Comment by cdimascio** [comment]
got it. yes, something like this can certainly be supported. i'll look into it. thanks

**Comment by cdimascio** [comment]
@sheldhur this is now implemented in express-openapi-validator@1.2.0.
Thanks for the suggestion!
